scripts/build.sh

```diff
@@ -17,5 +17,5 @@ mkdir -p build_cmake
 cd build_cmake
 
 core_count=`getconf _NPROCESSORS_ONLN`
-cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
+[ ! -f ../build_cmake/CMakeCache.txt ] && cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
```

Comment:
I'm not sure if this is something we should do. A couple of things :
1. I guessed the cause of the rebuild is the CBLEdition.h re-generation when calling cmake again. Can you check or confirm this?

2. If # 1 is true, I feel like we do not need to fix that. Developers can call make themselves if they don't want this to happen. The build.sh script is just for convenience.

Reply:
Exactly. `CBLEdition.h` does re-generated (contains timestamp) when calling cmake. So whenever cmake is not called, the build is fast enough - no problems found.

Reply:
I did edit `build.sh` as that was used within the ticket and in order to guarantee that we won't open another ticket in the future for this... I did put the check for `CMakeCache.txt` and only force run cmake when that file doesn't exist.

If this is not desired, I believe the ticket can be closed as there wasn't any issue found with the build.

Reply:
Will discard as this script is intended to be used by external to build lib and should only be used on initial build. Afterwards, it can be manually re-build using only `make`